scripts/cxg_admin.py

```diff
@@ -185,6 +185,31 @@ def create_cxg_artifacts(ctx):
                     s3_uri=s3_uri,
                 )
 
+@cli.command()
+@click.pass_context
+def migrate_schema_version(ctx):
+    """
+    Populates `schema_version` for each existing dataset. Since the schema version only exists
+    in the cxg file and we don't want to open them, we will call the cellxgene explorer endpoint
+    which contains the version. This is a one-off procedure since new datasets will have
+    the version already set.
+    """
+
+    import requests as rq
+    with db_session_manager() as session:
+        click.confirm(
+            f"Are you sure you want to run this script? It will assign schema_version to all the "
+            f"datasets",
+            abort=True,
+        )
+        for record in session.query(DbDataset):
+            dataset_id = record.id
+            url = f"https://api.cellxgene.cziscience.com/cellxgene/e/{dataset_id}.cxg/api/v0.2/config"
```

Comment:
annoyingly this will miss some datasets that were named early on with a different naming scheme (cellxgene.cziscience.com/cellxgene/e/{datasest_name}.cxg) but you should be able to do something like f"api.{record.explorer_url}/api/v0.2/config" getting the explorer url off the dataset

Reply:
Ah, I didn't know the endpoint had more formats. I will implement what you suggested!

Reply:
Also if possible to allow for testing I would set it up to build the api based on the passed in deployment stage. For staging it should be `https://api.cellxgene.staging.single-cell.czi.technology`

Reply:
I added the hostname from the explorer_url, that should work.